### ISSUE TITLE
Blogs redesigns - /learn /go /webinars /blog

### ISSUE THREAD

**Issue by Noemish** [Automattic/wp-calypso#71153]
The intention is to start a project where we work on all the blogs as a whole update and explore to unify some of them

/learn Blog is intended to be unified with the current /webinars.
[P2 Thread
](pdyEEc-hJ-p2)

@Noemish already worked on some explorations to unify /blog and /go some time ago and also the [educational portal ](mcyEC7EO76AgnD8nzP2yBI-fi-0%3A1&t=rSdazsFOwPAgX9JV-1)

**Comment by Noemish** [comment]
Relevant previous work that can be useful:

**Comment by ebinnion** [comment]
Checking in here, is this still a project that we're prioritizing? If so, are you doing this @Noemish or is someone else?

**Comment by Noemish** [comment]
@ebinnion Apologies, I've just seen this here. Unfortunately, this is not a project I'll be working on anymore (although I requested it) :(
I think @vinimotaa is working on /learn

**Comment by vinimotaa** [comment]
Yes, I'm working on /learn and at some point I believe we will incorporate /blog and /go into it as part of the roadmap. Sharing today progress on /learn @ebinnion

**Comment by ebinnion** [comment]
All good. Thank you for the clarification.
